frontend/src/components/forms/AgentChatWidget.module.css

```
@@ -0,0 +1,233 @@
+
+.assistantMessage {
+  align-self: flex-start;
+  background: var(--bg-section);
+  border: 1px solid var(--border);
+  color: var(--text-primary);
+}
+
+.messageText {
+  margin: 0;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)